scss/reset/_headings.scss

```diff
@@ -66,13 +64,29 @@ the tag as a heading.
 .text--display2 {
 	@extend %text--display2;
 }
-.text--display2 {
```

Comment:
That class was in there twice, so I didn't actually delete anything, just remove duplicate class